java/pre_package_concord.bash

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash --login
 set -e # fail fast
 git_root=$(git rev-parse --show-toplevel)
-target_dir=$git_root/jvm/target/concord
-getting_started=$git_root/jvm
+target_dir=$git_root/java/target/concord
```

Comment:
are you proposing we move all the folder jvm to java? 

Isn't this incorrect w/ the current folder layout?